tests/lib/dbus.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+DISPLAY=:0
+
+# shellcheck source=tests/lib/pkgdb.sh
+. "$TESTSLIB/pkgdb.sh"
+
+init_dbus_env(){
+    export $(cat dbus.env | xargs)
+}
+
+start_dbus_loop_unit(){
```

Comment:
The command in the snaps providing dbus names always does the same thing, instantiate the provider class and then start the glib main loop, see https://github.com/snapcore/snapd/pull/3484/files#diff-90d136194510bd1c3d9cf8a982e82728R28 or https://github.com/snapcore/snapd/blob/master/tests/lib/snaps/test-snapd-dbus-provider/provider.py#L23, the function itself just wires this into system-controlled services (either systemd or upstart based).